plugins/kubernetes/app/models/kubernetes/client_config_file.rb

```diff
@@ -0,0 +1,126 @@
+module Kubernetes
+  class ClientConfigFile
```

Comment:
FYI, there's currently a PR open for kubeclient ([PR #127](https://github.com/abonas/kubeclient/pull/127/files)) to implement this as part of the client gem. Once that gets merged in, I'll just use that functionality.